tools/content/model.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:io';
+import 'dart:convert';
+
+import 'package:json_annotation/json_annotation.dart';
+
+/// A data structure representing a message.
+@JsonSerializable()
```

Comment:
It looks like this `JsonSerializable` (and so the `json_annotation` import) doesn't end up getting used.